src/walletConnectV2Provider.ts

```diff
@@ -881,6 +853,24 @@ export class WalletConnectV2Provider {
     }
   }
 
+  private getConnectionParams(
+    options?: ConnectParamsTypes
+  ): EngineTypes.FindParams {
+    const methods = [...Object.values(Operation), ...(options?.methods ?? [])];
```

Comment:
Perhaps define a const array in `constants` that lists the default methods, instead of calling `Object.values(Operation)` here? Or, instead, perhaps define a temporary variable called `defaultMethods` above?

Reply:
Updated as recommended ( const array in constants )